ouroboros-network/src/Ouroboros/Network/ErrorPolicy.hs

```diff
@@ -54,11 +54,13 @@ import           Ouroboros.Network.Subscription.PeerState
 data ErrorPolicy where
      ErrorPolicy :: forall e.
                       Exception e
-                   => (e -> SuspendDecision DiffTime)
+                   => (e -> Maybe (SuspendDecision DiffTime))
+                   -- ^ Value of 'Nothing' means "trace and disconnect". It
+                   -- is equivalent to not having the policy at all.
```

Comment:
But it does not mean that, it means fall through to the remaining policies in the list.

There is no 4th disconnect option. It's just evaluating a list of handlers in order.

Reply:
@edsko @dcoutts is right here, we're falling through all the handlers and `sconcat` all the results.

Reply:
Yeah, I fixed that (force pushed).

Reply:
(Using Duncan's suggested wording now.)